<?xml version="1.0"?>

<launch>
	<arg name="robot_name" default="thorvald_001"/>
	
	<!-- Launch movement -->
	<group ns="$(arg robot_name)">
		<arg name="navigation_targets" default="$(find assessment_package)config/navigation_targets.yaml"/>
		<node name="navigator" pkg="assessment_package" type="navigator.py" output="screen" args="$(arg robot_name) $(arg navigation_targets)"/>
	</group>

</launch>
